kubernetes/deployment.tmpl

```diff
@@ -112,6 +112,56 @@ spec:
               value: staging
             - name: REDIS_URI
               value: "redis://hamlet-staging-redis"
+          volumeMounts:  # ??? Is this necessary?
+            - name: static-assets
+              mountPath: "/static"
+
+        - name: hamlet-staging-nginx
+          image: zooniverse/apps-nginx:xenial
+          ports:
+            - containerPort: 80  # ??? could this conflict with the Django app?
+          volumeMounts:
+            - name: static-assets
+              mountPath: "/static"
+      volumes:
+        - name: static-assets
+          configMap:  # ??? use EmptyDir: {} instead of the ConfigMap?
+            name: hamlet-nginx-conf-staging
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  creationTimestamp: null
+  name: hamlet-nginx-conf-staging
+data:
+  nginx.conf: |+
+    upstream docker-hamlet {
+      server localhost:81;
```

Comment:
I'm not entirely sure about this one. Is this saying, _"Hey, there's actually another server on this network, and we'll call it `docker-hamlet` so we can access it via `http://docker-hamlet`, and that server (the running software) is found at localhost, port 81"_

Er, if that's so, I might need to change this to `8080` because that's where I believe the Django app lives.